lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/DBOAccessRequirementDAOImpl.java

```diff
@@ -424,4 +375,32 @@ public Void mapRow(ResultSet rs, int rowNum) throws SQLException {
 		});
 		return stats;
 	}
+
+	@WriteTransactionReadCommitted
+	@Override
+	public AccessRequirement updateVersion(Long accessRequirementId) {
+		DBOAccessRequirement current = getAccessRequirementForUpdate(accessRequirementId);
+		Long newVersion = 1L;
+		if (current.getCurrentRevNumber() != null) {
```

Comment:
Do not expect nulls so throw exception if that occurs so we can figure out what happened.